### ISSUE TITLE
pre-trained model is not updated after a bug in loss fixed

### ISSUE THREAD

**Issue by amsword** [zylo117/Yet-Another-EfficientDet-Pytorch#273]
The pre-trained model is based on the code before this bug fix: https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/commit/8e695273398e6f3069183f70c1cb3563bf8954ed

When i run the efficientdet_det.py, the latest code does not seem to be good. If i revert the change, the results look good.

**Comment by zylo117** [comment]
No, it's not related. you just need to set `force_input_size=1920`. Actually I've mentioned it on readme though, right above the demo image.
